src/main/java/com/aws/greengrass/clientdevices/auth/certificate/CertificateStore.java

```diff
@@ -154,42 +151,25 @@ public PrivateKey getCAPrivateKey() {
     /**
      * Get certificate chain using private and certificate URIs.
      *
-     * @param privateKeyUri  private key URI
      * @param certificateUri certificate URI
      * @return X509Certificate list that corresponds to the given private key and certificate URIs
-     * @throws KeyLoadingException              if it fails to load key
      * @throws CertificateChainLoadingException if no certificate chain is found
-     * @throws ServiceUnavailableException      if the security service is not available
+     * @throws URISyntaxException               if provided uri scheme is not file
      */
-    public X509Certificate[] loadCaCertificateChain(URI privateKeyUri, URI certificateUri)
-            throws CertificateChainLoadingException, KeyLoadingException, ServiceUnavailableException {
-        KeyManager[] km = securityService.getKeyManagers(privateKeyUri, certificateUri);
-
-        if (km.length != 1 || !(km[0] instanceof X509KeyManager)) {
-            throw new CertificateChainLoadingException("Unable to find the X509 key manager instance to get the "
-                    + "certificate chain using the private key and certificate URIs");
-        }
-
-        X509KeyManager x509KeyManager = (X509KeyManager) km[0];
-        KeyPair keyPair = securityService.getKeyPair(privateKeyUri, certificateUri);
-        String[] aliases = x509KeyManager.getClientAliases(keyPair.getPublic().getAlgorithm(), null);
-        if (aliases == null) {
-            throw new CertificateChainLoadingException(
-                    "Unable to find aliases in the key manager with the given private key and certificate URIs");
+    public X509Certificate[] loadCaCertificateChain(URI certificateUri) throws CertificateChainLoadingException,
+            URISyntaxException {
+        if (!Objects.equals(certificateUri.getScheme(), fileScheme)) {
+            throw new URISyntaxException(certificateUri.toString(), "Invalid URI - certificateUri only supports file "
```

Comment:
Only? So even if I don't have a chain, I only have 1 cert, I must provide it on disk?